specs/credentialmanagement/index.src.html

```diff
@@ -821,9 +867,10 @@ <h4 id="interfaces-request-options">
 
     <pre>
       navigator.<a idl>credentials</a>.<a idl lt="get()">get</a>({
-        "<a idl>types</a>": [ "federated" ],
-        "<a idl>options</a>": {
-            "<a idl>providers</a>": [ "https://identity.example.com/" ]
+        "<a idl>filters</a>": {
+            "federated" : {
+                "<a idl>providers</a>": [ "https://identity.example.com/" ]
```

Comment:
I agree with the general suggestion that we need both a list of types we care about, and a list of options associated with each of those types. Perhaps a slight shift to place the two things in two separate arguments might make sense? That is:

Simple:
```
navigator.credentials.get(['password'])
```

Marginally complex:
```
navigator.credentials.get(
    ['password', 'federated'],
    {
        'federated': {
            'providers': [ ... ]
        }
    });
```

Complex:
```
navigator.credentials.get(
    ['password', 'federated', 'example'],
    {
        'federated': {
            'providers': [ ... ]
        },
        'example': {
            'property': true,
            'life': 43,
            'lifeOffset': 1
        }
    }
```

I don't really like this either, to be honest, but I don't have any better ideas.

It's not clear where `suppressUI` goes in this model, either. Perhaps another top-level `global options` argument? That gets ugly pretty quickly.